### PR TITLE
Add representative signature on review page

### PR DESCRIPTION
**Pull request by LinnJS**
## Description
On the 10-10CG update the signature boxes on the review page to include signing as a representative 

## Testing done
- Manual testing

## Screenshots
image

## Acceptance criteria
- [x] We allow user to sign as representative on review page if they choose to do so
- [x] The validation allows user to sign as POA 
- ~~ [ ] Send the signatures to BE so they are stored and can be used on PDF ~~ (New ticket is made for this task [here](https://app.zenhub.com/workspaces/vsa---caregiver-5fff0cfd1462b6000e320fc7/issues/department-of-veterans-affairs/va.gov-team/22819))
- ~~ [ ] BE accepts data and data is transformed correctly according to feature flag ~~ (Same)
- [x] Signature page work is feature flip properly
- [x] RE matches designs below

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/caregivers/components/PreSubmitInfo/SignatureInput.jsx [hunk 1]
```diff
@@ -49,21 +53,33 @@ const SignatureInput = ({
...
-      // show error if there is a form error and has not been submitted
+      /* show error if user has touched input and signature does not match
+         show error if there is a form error and has not been submitted */
       if ((isDirty && !signatureMatches) || (showError && !hasSubmit)) {
         setIsSigned(false);
         setError(true);
       }
 
-      if (isDirty && signatureMatches) {
+      if (
```
Comment (LinnJS) [inline src/applications/caregivers/components/PreSubmitInfo/SignatureInput.jsx:66]: Need to add comment explaining additional logic

**Review by Mottie — APPROVED** [review]

**Review by craigwheeler — APPROVED** [review]

File: src/applications/caregivers/components/AdditionalInfo/index.jsx [hunk 2]
```diff
@@ -275,3 +275,30 @@ export const RepresentativeAdditionalInfo = () => {
...
+export const RepresentativeDocumentUploadDescription = () => {
+  return (
+    <section>
+      <h3 style={{ padding: 0, marginBottom: '1.3em' }}>
+        Upload your supporting documentation
+      </h3>
+
+      <p>
+        You will first need to scan a copy of your document onto the same device
+        that you are submitting your online application (i.e. computer or mobile
```
Comment (Mottie) [inline src/applications/caregivers/components/AdditionalInfo/index.jsx:288]: Not to be a grammar cop, but this should be `e.g.` not `i.e.`... and how does one scan a document into a mobile phone?

**Review by Mottie — APPROVED** [review]
ghe-emoji

File: src/applications/caregivers/components/AdditionalInfo/index.jsx (hunk 2 above)
Comment (LinnJS) [reply src/applications/caregivers/components/AdditionalInfo/index.jsx:288]: Roger that, I will pass this to @Dene-human our new designer on Caregiver for feedback, this is not the final copy so will get it in the next run.

Comment (Dene-human) [reply src/applications/caregivers/components/AdditionalInfo/index.jsx:288]: I certainly agree @Mottie. This is something that is directly from the project team that we need in our research study, but is definitely not final content. Thank you for checking!

**Pull request merged**